src/components/Guilds/Sidebar/StakeTokens.tsx

```diff
@@ -87,14 +93,46 @@ const ButtonLock = styled(Button)`
 
 export const StakeTokens = ({ onJoin }) => {
   const [dxdValue, setDXDValue] = useState(0);
+  const {
+    data: { timeForExecution, proposalTime, name, token },
+    error,
+  } = useConfig();
+
   return (
     <GuestContainer>
       <DaoBrand>
         <DaoIcon src={dxIcon} alt={'DXdao Logo'} />
-        <DaoTitle>DXdao</DaoTitle>
+        <Loading text loading={!name && !error}>
+          <DaoTitle>{error ? 'DxDao' : name}</DaoTitle>
+        </Loading>
       </DaoBrand>
       <InfoItem>40% Quorum</InfoItem>
-      <InfoItem>6 days proposal duration</InfoItem>
+      <Loading text loading={!token && !error}>
+        <InfoItem>
+          {!error && (
+            <>
+              Token: {isAddress(token) ? shortenAddress(token) : ''}

```

Comment:
We should resolve the token name at some point also but for now this is good

Reply:
I just added it because `shortenAddress` throws an error if isn't a valid address.